pattern-1/bosh-release/jobs/wso2is_2/spec

```diff
@@ -34,16 +34,16 @@ properties:
     description: wso2is address
 
   wso2is.mysql.host:
-    description: wso2is mysql hostname
+    description: Hostname/IP of the MySQL server in which WSO2 Identity Server product database resides
```

Comment:
Let's rename the key to be not MySQL specific here, so that we can use them for other databases as well.

Reply:
@DilanUA this has been fixed in PR https://github.com/wso2/pivotal-cf-is/pull/44.